docs/src/api/class-pageerror.md

```diff
@@ -0,0 +1,62 @@
+# class: PageError
+* since: v1.38
+
+[PageError] class represents objects created by context when there are unhandled
+execeptions thrown on the pages and dispatched via the [`event: BrowserContext.pageError`] event.
+
+```js
+// Log all uncaught errors to the terminal
+context.on('pageerror', pageerror => {
+  console.log(`Uncaught exception: "${pageerror.error()}"`);
+});
+
+// Navigate to a page with an exception.
+await page.goto('data:text/html,<script>throw new Error("Test")</script>');
+```
+
+```java
+// Log all uncaught errors to the terminal
+context.onPageError(pagerror -> {
+  System.out.println("Uncaught exception: " + pagerror.error());
+});
+
+// Navigate to a page with an exception.
+page.navigate("data:text/html,<script>throw new Error('Test')</script>");
+```
+
+```python async
+# Log all uncaught errors to the terminal
+context.on("pageerror", lambda pageerror: print(f"uncaught exception: {pageerror.error}"))
+
+# Navigate to a page with an exception.
+await page.goto("data:text/html,<script>throw new Error('test')</script>")
+```
+
+```python sync
+# Log all uncaught errors to the terminal
+context.on("pageerror", lambda pageerror: print(f"uncaught exception: {pageerror.error}"))
+
+# Navigate to a page with an exception.
+page.goto("data:text/html,<script>throw new Error('test')</script>")
+```
+
+```csharp
+// Log all uncaught errors to the terminal
+context.PageError += (_, pageerror) =>
+{
+  Console.WriteLine("Uncaught exception: " + pageerror.Error);
+};
+```
+
+## method: PageError.page
+* since: v1.38
+- returns: <[null]|[Page]>
```

Comment:
If it's a page error it implies that there is a page where the error was thrown, let's change it to be non nullable?

Reply:
Never mind, I see it was suggested [above](https://github.com/microsoft/playwright/pull/24452#pullrequestreview-1557970923), I'll discuss this with Dmitry.